constants/networks/Arbitrum.ts

```diff
@@ -0,0 +1,10 @@
+import { Network } from "../../test/helper/interfaces/network"
+
+export const Arbitrum = function (): Network {
+  return {
+    name: "arbitrum",
+    // url: `${process.env["ARBITRUM_ARCHIVE_NODE_URL"]}`,
```

Comment:
let me send you the Arbitrum chainstack address we use, those public nodes tend to be not so reliable for testing